frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.spec.ts

```diff
@@ -25,11 +25,13 @@ describe('Namespace', () => {
     listPage.rows.shouldNotExist(newName);
     listPage.filter.byName(testName);
     listPage.rows.shouldExist(testName); // created via cy.createProject(testName) above
+    cy.testA11y(`Namespace List page`);
```

Comment:
Unnecessary backticks. (I thought we had an eslint rule for this.)

```suggestion
    cy.testA11y('Namespace List page');
```